en/flight_controller/raspberry_pi_navio2.md

```diff
@@ -176,87 +68,61 @@ Make sure the Pi has network access, and you can SSH into it.
 
 ```sh
 echo "Hello" > hello.txt
-scp hello.txt pi@px4autopilot:/home/pi/
+scp hello.txt pi@navio.local:/home/pi/
 rm hello.txt
 ```
 
-This should copy over a "hello.txt" file into the home folder of your RPi.
+This should copy over a "hello.txt" file into the home folder of your Pi.
 Validate that the file was indeed copied, and you can proceed to the next step.
 
 ## Building the Code
 
-Either build the source code on your development computer ("cross-compiler" build) or build it on the RaPi ("native" build) as shown below.
+Follow the instructions below to build the source code on your developmenet machine (Ubuntu 18.04) and transfer the compiled program to the Pi, unfortunetealy, the option to build the code natively(on the Pi) is no longer available.
 
 ### Cross-compiler Build
 
-First install the [standard developer environment on your Ubunto development computer](../dev_setup/dev_env_linux.md).
+First install the [standard developer environment on your Ubunto development computer](../dev_setup/dev_env_linux_ubuntu.md#raspberry-pi).
 
-Set the IP (or hostname) of your RPi using:
+Specify the IP (or hostname) of your Pi using:
 
 ```sh
-export AUTOPILOT_HOST=192.168.X.X
+export AUTOPILOT_HOST=navio.local
 ```
 
 or
 
 ```sh
-export AUTOPILOT_HOST=pi_hostname.domain
+export AUTOPILOT_HOST=192.168.X.X
 ```
 
 :::note
-The value of the environment variable should be set before the build, or `make upload` will fail to find your RPi.
+The value of the environment variable should be set before the build, or `make upload` will fail to find your Pi.
```

Comment:
I am used to seeing RPi or RasPi. I don't particularly care as long as the naming is consistent through the document.

```suggestion
The value of the environment variable should be set before the build, or `make upload` will fail to find your RPi.
```